spec/lib/account_spec.rb

```diff
@@ -0,0 +1,101 @@
+describe BankAccount::Account do
+
+  describe '.new(id, initial_balance)' do
+    it 'responds' do
+      expect(BankAccount::Account).to respond_to :new
+    end
+
+    context 'when instantiating an accepted account' do
+      before :all do
+        @my_account = BankAccount::Account.new(1,125)
+      end
+
+      it 'is part of Account class' do
+        expect(@my_account).to be_a BankAccount::Account
+      end
+
+      it 'has an id' do
+        expect(@my_account.id).to eq(1)
+      end
+
+      it 'has a balance' do
+        expect(@my_account.balance).to eq(125)
+      end
+
+      it 'has an initial_balance' do
+        expect(@my_account.initial_balance).to eq(125)
+      end
+    end
+
+    context 'when a negative account balance is passed in' do
+      it 'raises an ArgumentError' do
+        expect { BankAccount::Account.new(1, -1) }.to raise_error(ArgumentError)
+      end
+    end
+  end
```

Comment:
Excellent use of `raise_error` :+1: If you want to be extra fancy, you can also [test for the error message itself](https://www.relishapp.com/rspec/rspec-expectations/v/3-2/docs/built-in-matchers/raise-error-matcher#match-class-+-message-with-string):
`expect { BankAccount::Account.new(1, -1) }.to raise_error(ArgumentError, "insufficient opening balance!")`